plugins/module_utils/crypto/cryptography_openssh.py

```diff
@@ -0,0 +1,484 @@
+# -*- coding: utf-8 -*-
```

Comment:
About the placement of this file: I think it would be better to have a separate directory, `plugins/module_utils/openssh/`, instead of cramming more content into `plugins/module_utils/crypto/`.

Reply:
Moved the file to plugins/module_utils/crypto and incorporated the remaining suggestions from your review.
Thank you.